test/1.4/expressions/T_static_export.dml

```diff
@@ -0,0 +1,55 @@
+/*
+  © 2022 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+
+device test;
+
+import "testing.dml";
+
+method foo(uint64 a) -> (uint64) {
+    return a + 1;
+}
+
+bank b {
+    method m() -> (uint64) {
+        return 10;
+    }
+    register r size 4 @ 0 {
+        method m() -> (uint64) {
+            return 5;
+        }
+        field f @ [0:0] {
+            method m() -> (uint64) {
+                return 1;
+            }
+        }
+    }
+}
+template m_holder {
+    method m() -> (uint64) {
+        return 3;
+    }
+}
+
+is m_holder;
+
+independent method indie(uint64 i) -> (uint64) {
+    return i + 7;
+}
+
+method test() throws {
+       expect((&foo)(dev.obj, 1) == 2,
+              "trivial method with argument and return");
+       expect((&b.m)(dev.obj) == 10,
+              "method statically exported from bank");
+       expect((&b.r.m)(dev.obj) == 5,
+              "method statically exported from register");
+       expect((&b.r.f.m)(dev.obj) == 1,
+              "method statically exported from field");
+       expect((&m)(dev.obj) == 3,
+              "method accessing template parameters");
```

Comment:
The methods don't use the implicit dev arg, so the test doesn't prove that the static trampoline converts the arg correctly.

Reply:
Neither do the tests in `T_export`. Same thing applies there?

Reply:
I suppose so.

Reply:
Both now access dev by mutating a session.